packages/expo/static/loading-page/index.html

```diff
@@ -0,0 +1,274 @@
uxf+bO+P3AuNt8E8tobTfveSKvGUERoql1nNu+yj8j/vjgnPn/xryXAvMdvavobumo7e5770urr2u6bfvR38GOFfEeUpHz/tePtyi/0fhv8FZ8ZKrNLNcXg+1QZS2JYq/eaTxPVue/96bxj113hqWzgrkke63bLW8YznYQrz9sd6G/dC5+9AR9P3vZ8Zozj20f1v0e9738miLnkvLHIX/0hHegO8lVwlTLi+v8JVy16f4G8i+drR8R/Hn4nwObwSQGcyhMkv8uGt+tn3Op0FlPvS8eejc7dl3yXHf9352jObcJW2/FLbn/WguXcqmLbhH4d8StQvLeMlVqiyXV6v4lfLXo/gbyL769/adbxCwqNne1+IDba8I6DnaexN+jLm7V1x93/u7U7kz7WXRw+N43u+dc9fw8tEWfgTnuSVYnhCl93sXv1zf9p6/gWzHq0l6/nsLVcuXUZSzZWWwLipa7mjfTgTsHHkbsZcH3txrr1QaaJ5Sq/cAAAc5SURBVPLubS9vd1eyAdD334tkAu/FNHE+vlbNcy/fbt27R134dXCfasLVkk377rfLiDyuDcb3agtVf4+aP4gCnDJCW9BvTwZ+MW9GfdT8bN+eZ+dZpP3id+zPQxyTflzrTkep799ofvWRbpNjJBpH4b/Q3OkzsJqKp5qAI4nnx6SMyOPagHyv9qZ6MdH5Zs8m99Z/4J0fATtH3kbsn72er3sEwcuxEblL8/gxmdtxyl3m0lZfF38kGl6+XdfKeW3HGf98zK4qMZJ4vkHlknxcG5Dv1d7UWkxY/G1R2Vv/gXd+BPQc1TZhv8idr3sEoZZjI7KX5PFjMrfjlLvMJS2+PvZINGo5d31rxzUehX88Vg/COZJ4vmHlkrzVDci3v031N622zCVGz4npc3hzsdem+BTm+xL5dw6mH9O5HafcZc6x6PZkR6KxV45f0/tHUfgREG6SteA85QQcST4/LiYq0jUUX+yRUP2N67rGH4W/He+9NsXHPtftKF1/NOwpxcZSEJqGPaW9pOZoLyJ75XdN/6Xoj6bw2wD03giA/zkkJuPSS9DIlyJyTsGizoc+PvZiwBzG5sH2Q8d0b/17bYyPfa73jutWvDyO/V0j7RhbNT4euX40pKY8hY1TpuQo/I8nL5uWjiStfiv0FPI338Sa4bnY4FOI48WCsyPwMdf7BDOPY3/XOAp/Hvej8OfxuHpv5AzpOSUtJmBqGUtwHnt88k3s6ikYFB6F/zpxP+b6/DiH/SHbJLJOoeCx7w+FQx1COxpR+Cj8nSBec/h4E1BGeySJSym11J/AGwPt3y0UDm0P2vH5Amgh7voJA6A9/lfMJpVTN+TSU3vD5uf3/C5wm7N1ucQZidBTKfY6io/2Ur92Ikyen/kr25HQaygajTxKT2lz7KRHIyaXG9IPFoKWp1b6r/mQnNlZekq5HXLHrWAusRmqfAdosj6JwW6EJFGeYkyOwv8k0rd0opvQpYhQ8hR/SpvjLRZ+dwoO4lUi8JRyGwHz83t+F8h3gKtMxYMq6UboKPwPOj9N5WHy/Mxf5Y6EXkPRaORRemqbY8Px4aFOmg3jHIz7RODI0RhHPy+7Za2YhHwHKIafHKEboaPw3/ac9z7nPxJ6dP5SpI5N1Y+Zv8n6vAf1chE48jPGNp742Dh3S5oVCP20+t3hJ0fsRekpfr6PSXwSl/q9bOy9EQjOe4LPhNZLeD8MzrYgJIfqiz8R6lH4b2Min0vh9ws75mB+FT+3taozdSRaT7XQ6zig/XQLP7zr7NDHIkCQZl9l1J7LBsxIddKKbMfxwhF4Lnnn59tIGSsnoFy9Jc9TpXQjJgn1XOJzFP6nmuUdv7qLoCpfLo3nsgEzJP5GzNHjeK0IPJe88/Nt2wouV++1Zuvh9XQjdhT+h5+kPS04LvvPR7O7SFzI/rbyXDZrNzwXJPrF4XyFx3ydH8MeQlhr7oJziS5cf+W5Yk+WOB45uez9DJP8yZ7x24zuFf9j4eQRm1k4uWQ7ks9wjeXhuWBv7+J/zNUFJ0tB+/M2twLbq04peybN0eg9x6KPFDgK/7IQjoWT7wijCyeXQq8dyaOYlBHbi+IXkO3ox1xtj92MpD9vcyuwvepmrHkavKPROwr/05jvqhfHGX81NO7A6MIphdtb0FFMyojtRfELyHb0Y662x25G0p+3uRXYXnUz1jwN3tHoHYX/acx31Yuj8FdD4w6MLpxSuL0FHcWkjNheFL+AbEc/5mp77GYk/XmbW4HtVTdjzdPgHY3eUfifxnwPe9F7IwCgYzH1wzm6wBJSO6pHsUmROlqPLwJ+Eff8mFs57VXj4T8/2khEn2uht9nwbD7jt46j3yv+x2LzolbSRhZckupH9Sj+KVpH63FF4BKFv79iHleMLmVtbx86in6K/FH4UyyK1rHgipC4hN6Cy4X6UT0Kfx6xo/d4InAU/oebq94+dBT+NDdH4U+xcFv9MuWKPStib8GVwWhH9Sj8ZcQOyu1H4BJFH163V8vtx+UaFo7sQUfhTzPxrAt/CkP/sj95j0XISGw7jixQH7kd+ePNgh+1g3peBPrFfFtGt7P5PJufg3Qv6keRb2fBUfiX+PQ+72cYjwXLSGw79hZsHbUd+aPw1yN3jGyPwFH4t8fukpK9feQo/I3o3718Pg/waYQhDB2FvxehfcZ7C7au5Sj89dgcI5eKwFH4LxXZ83B7+8hR+CvxlTOkN5/45FH4GZ6j8DMSlz32Fmxd+1H467E5Ri4VgaPwXyqy5+H29pGj8DvxRdG/e+v05hvfPAq/Ds9R/HU0LtfuLVpfc7vwQ+a43O9H7qBui0C/6AN3Ppv7mbzN3uciNRLxo/CbbJDL+2/e+fjpzVe/dnrz7e8ehd+EJ+uOvhHIhKRzLGwbkbn+yMIeQ5ybieONw1hUHwPXWNG2npyfeXMZZ/UffURg6ywcxb6eP2/efuf05vPvnu7/x//p9OI73z39/6rYW0vPBRr9AAAAAElFTkSuQmCC"/>
```

Comment:
@jonsamp this is the placeholder Form Duo icon (I think?). Is there an SVG version or literally any small placeholder we could put here instead of this massive string that got inlined from https://github.com/expo/expo-cli/blob/ebe4a9c987420a06d27a6fcccd42439ebe1b4cd3/packages/xdl/static/loading-page/index.html#L234?

<img width="727" alt="ohno" src="https://user-images.githubusercontent.com/379606/157996710-d5e74579-3708-4608-82f9-47e60094ab2f.png">